src/Resources/skeleton/command/Command.tpl.php

```diff
@@ -2,18 +2,29 @@
 
 namespace <?= $namespace; ?>;
 
+<?php if ($use_attributes && $use_command_attribute): ?>
```

Comment:
Do we need to check both? `$use_command_attribute` already checks for PHP 8.0 or higher. So it seems like it fully describes which situation we are in, right?

Reply:
I think we _could_ drop `$use_attributes` here, but then we would loose the ability to determine if attributes should be used based on the `platform-overrides`. Then again as I'm saying this, regardless of platform overrides, they wouldn't be able to use attributes anyway if PHP was not 8+ at runtime.. 

Thinking we drop `$use_attributes` for now and when I'm able to accomplish the helper in `PhpCompatUtil`, we can re introduce / refactor our logic here.

Reply:
Ah, you're right about the `config.platform.php` thing. So then... shouldn't we keep the `$use_attributes`? Otherwise, it's possible that I'm using PHP 8... but my `config.platform.php` is set to 7.4... and suddenly we generate code that uses attributes?